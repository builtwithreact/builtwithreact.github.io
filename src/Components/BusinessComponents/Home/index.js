import React, { Component } from 'react';
import SubHeader from './../../BaseComponents/SubHeader'
import './Home.scss';

class Home extends Component {
    render() {
        return (
            <main className="Home">
                <SubHeader />
            </main>
        );
    }
}

export default Home
