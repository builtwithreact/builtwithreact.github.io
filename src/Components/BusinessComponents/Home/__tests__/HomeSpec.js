import React from 'react';
import renderer from 'react-test-renderer';
import Home from './../index';
jest.mock('./../../../BaseComponents/SubHeader', () => 'SubHeader')


describe('Home', ()=>{
    it('should render correctly', ()=>{
        let tree = renderer.create(
            <Home />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
