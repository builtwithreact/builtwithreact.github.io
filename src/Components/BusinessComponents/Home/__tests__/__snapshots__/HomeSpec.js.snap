// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly 1`] = `
<main
  className="Home"
>
  <SubHeader />
</main>
`;
